### ISSUE TITLE
Set SEQUENCE on Event

### ISSUE THREAD

**Issue by justin1364** [C4ptainCrunch/ics.py#115]
I'm trying to compose a REPLY and it looks like [SEQUENCE is required](https://tools.ietf.org/html/rfc2446#section-3.2.3). It looks like `ics.Event()` doesn't support this field?

```
c.method = "REPLY"
e = ics.Event()
e.uid = "original_id"
e.sequence = 1
c.events.append(response_event)
print c

BEGIN:VCALENDAR
PRODID:-//POD/BoardCalendar//EN
VERSION:2.0
METHOD:REPLY
BEGIN:VEVENT
DTSTAMP:20180306T170833Z
UID:1
END:VEVENT
END:VCALENDAR
```

**Comment by justin1364** [comment]
https://github.com/Podfactor/ics.py/commit/1e8731fa2714cc58369e54a7261c236851e2e47a might be all you need. happy to submit a PR if you'd like.

**Comment by C4ptainCrunch** [comment]
Indeed, we do not support `SEQUENCE`. But unfortunately, supporting it is more complicated than the commit you referenced.

We can see in https://www.kanzaki.com/docs/ical/sequence.html
> When the "Organizer" makes changes to one of the following properties, the sequence number MUST be incremented: [..]

So we would need to have a way to known if we are opening the calendar as organizer and then also increment the `SEQUENCE` automatically when needed.

**Comment by JMMarchant** [comment]
Whilst automating support for `SEQUENCE` is perhaps impossible, it should be noted that the MUST requirement is no longer present in RFC5545 (https://icalendar.org/iCalendar-RFC-5545/3-8-7-4-sequence-number.html) instead deferring to: 

> It is monotonically incremented by the "Organizer's" CUA each time the "Organizer" makes a significant revision to the calendar component.

Given this change, can support for parsing and interacting with the sequence number be added with the caveat that incrementing/changing it it is the end-users responsibility?

**Comment by C4ptainCrunch** [comment]
> Given this change, can support for parsing and interacting with the sequence number be added with the caveat that incrementing/changing it it is the end-users responsibility?

I agree
